soc/nordic/nrf54h/power.c

```diff
@@ -141,5 +138,8 @@ void pm_state_set(enum pm_state state, uint8_t substate_id)
 
 void pm_state_exit_post_ops(enum pm_state state, uint8_t substate_id)
 {
+	if (state == PM_STATE_SUSPEND_TO_IDLE) {
```

Comment:
It turned out, forcing POWERON is not required in the Radio CPU.

```suggestion
	if (state == PM_STATE_SUSPEND_TO_IDLE && !IS_ENABLED(CONFIG_SOC_NRF54H20_CPURAD)) {
```